components/builder-depot/src/server.rs

```diff
@@ -2623,6 +2690,10 @@ mod test {
         package.set_config("config".to_string());
         package.set_target("x86_64-linux".to_string());
 
+        let mut channels = protobuf::RepeatedField::new();
+        channels.push("unstable".to_string());
+        package.set_channels(channels);
```

Comment:
So I tried something like this:

```
fn add_package_channels(package: &mut OriginPackage, channels: Vec<String>) -> &mut OriginPackage {
    package.set_channels(::protobuf::RepeatedField::from_vec(channels));
    package
}
...
package.set_target(...)
add_package_channels(&mut package, vec!("unstable".to_string()));
...
```

But somehow that feels less ideal than just something like:

```
package.set_channels(protobuf::RepeatedField::from_vec(vec!("unstable".to_string())));
```

... if not quite as DRY. Thoughts? The latter seems okay to me given the state of test setup in general, though I agree it does feel like we could reduce some duplication in this code more generally.